### ISSUE TITLE
Serialization of escaped quotes is broken

### ISSUE THREAD

**Issue by atextor** [atextor/turtle-formatter#9]
Strings containing escaped quotes are broken when serialized.
See https://github.com/atextor/owl-cli/issues/10 for original description.

**Comment by atextor** [comment]
Fixed in 1.2.8.

**Comment by VladimirAlexiev** [comment]
@atextor do you use triple quotes when the string has many special chars?
IMHO this is much nicer 
```ttl
"""Jesus said "Belhold!" 
And the crowd answered "We see""""
```
Than
```ttl
"Jesus said \"Belhold!\"\nAnd the crowd answered \"We see\""
```
